test_sets/Ideas_for_SDL_must_send_WARNINGS_success_true_to_mobile_app_if_HMI_respond_WARNINGS_at_least_to_one_component_of_RPC.txt

```diff
@@ -0,0 +1,29 @@
+./test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_TTS_SetGlobalProperties_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_SetGlobalProperties_UI_SetGlobalProperties_SUCCESS

```

Comment:
@vvvakulenko, scripts from row 1 - 16 can be performed in loop for all results codes success:true, based on clarification:
"Clarification regarding ...." for (WARNINGS, WRONG_LANGUAGE, RETRY, SAVED)

Description can be as example:
Checks_that_SDL_sends_WARNINGS_success_true_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_SetGlobalProperties_UI_SetGlobalProperties_loop_resultcodes_true